react-app/cypress/e2e/load.cy.ts

```diff
@@ -0,0 +1,5 @@
+describe('Load', () => {
+  it('should load the page', () => {
+    cy.visit('http://localhost:3000');
```

Comment:
I wish we could change the port using an env var (e.g. `ROMCAL_APP_PORT` or whatever env var name you prefer).

Reply:
asbefore, will do